docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -54,7 +54,7 @@ spec:
 | ---| ---| ---| --- |
 |``host`` | The host (domain name) of the server. Must be a valid subdomain as defined in RFC 1123, such as ``my-app`` or ``hello.example.com``. When using a wildcard domain like ``*.example.com`` the domain must be contained in double quotes.  The ``host`` value needs to be unique among all Ingress and VirtualServer resources. See also [Handling Host and Listener Collisions](/nginx-ingress-controller/configuration/handling-host-and-listener-collisions). | ``string`` | Yes |
 |``tls`` | The TLS termination configuration. | [tls](#virtualservertls) | No |
-|``gunzip`` | Enables or disables [decompression](https://docs.nginx.com/nginx/admin-guide/web-server/compression/) of gzipped responses for clients. Allowed values are: "on" or "off". If the ``gunzip`` value is not set, it defaults to ``off``.   | ``string`` | No |
+|``gunzip`` | Enables or disables [decompression](https://docs.nginx.com/nginx/admin-guide/web-server/compression/) of gzipped responses for clients. Allowed values are: "on" or "off". If the ``gunzip`` value is not set, it defaults to ``off``.   | ``boolean`` | No |
```

Comment:
If this is a bool, allowed values are `“true”/“false”, “yes”/“no” or “on”/“off”` AFAIK

Reply:
The question is if we keep only `on`/`off` to be consistent with [N docs](https://nginx.org/en/docs/http/ngx_http_gunzip_module.html), and [this one](https://docs.nginx.com/nginx/admin-guide/web-server/compression/). From user's perspective it will be either `gunzip: on` in VS or no gunzip field.

Reply:
I understand that, but the openAPI validation will still allow any of the allowed bool values. Sticking to "on"/"off" in the examples is definitely the right thing, but stating `Allowed values are: "on" or "off".` is technically not true here.

Reply:
Updated docs and pushed the change